plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list/payment-gateway-list.scss

```diff
@@ -35,6 +35,11 @@
 		&-inner {
 			padding: $gap-large 0;
 			align-items: flex-start;
+
+			@media screen and ( max-width: 782px ) {
+				display: grid;
+				grid-template-columns: 30px minmax(200px, 1fr) 40px;
```

Comment:
comment here would help

Reply:
I’ve added comments to explain the code.